### PR TITLE
update to include english devolved area checks

### PR DESCRIPTION
**Pull request by sarahmwong**
Including english devolved area name and code checks. Ran past the FE team and they are happy using these names as part of the wider push for harmonisation across releases.

### CODE REVIEW

**Review by rmbielby — APPROVED** [review]
Ran App and tests and all looked good.

**Comment by rmbielby** [comment]
@cjrace Needs one of the designated approvers to approve before this can be merged. Won't just let me step in to approve without an introduction!

**Review by cjrace — APPROVED** [review]

**Comment by cjrace** [comment]
@rmbielby I've approved now but will also add you to that group so you're an approved reviewer

**Pull request merged**
